src/app/server/Server.h

```diff
@@ -278,7 +278,10 @@ struct CommonCaseDeviceServerInitParams : public ServerInitParams
         }
 
         // Session Keystore injection
-        this->sessionKeystore = &sSessionKeystore;
+        if (this->sessionKeystore == nullptr)
+        {
+            this->sessionKeystore = &sDefaultSessionKeystore;
```

Comment:
Realistically: someone who wants to inject a different keystore should not be using the CommonCaseInit thing.  This is making them pay the flash cost for DefaultSessionKeystore even though they are not using it, right?

Reply:
That's right, thanks for pointing it. I thought that someone would inherit the base DefaultSessionKeystore, and in this case, it must be compiled anyway. But if it comes to injecting a completely new SessionKeystore implementation, the default one will be redundant, but compiled. I will remove this part from this PR.